test/IntegrationTests/AspNetTests.cs

```diff
@@ -1,3 +1,20 @@
+// <copyright file="AspNetTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if !NETFRAMEWORK
```

Comment:
#if NETFRAMEWORK?

Reply:
See AspNetTests. I kept the existing behavior

Reply:
Good catch @rajkumar-rangaraj. This test would have previously run with a NET462 build of the test project. However, the test app itself will still run as a .NET Framework app. The container filter is preventing this test from being run on Linux and MacOS.  By using `!NETFRAMEWORK` instead of `NETFRAMEWORK` I think that we are actually running this test twice: once for `netcoreapp3.1` and once for `net6`.

Reply:
You are right. It should be NETFRAMEWORK

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/878/commits/bc2c8c7d0e4f4f8e78b964614826abbd783f200a